app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -122,6 +126,45 @@ class ChatActivity : AppCompatActivity(), IChatView {
         compensateTTSDelay()
     }
 
+    fun openChatSearch(view: View) {
+        if (isSearching == false) {
+            chatSearchInput.setVisibility(View.INVISIBLE)
+            hideSoftKeyboard(this, window.decorView)
+        } else {
+            chatSearchInput.setVisibility(View.VISIBLE)
+            handleSearch()
+        }
+    }
+
+    fun handleSearch() {
+        chatSearchInput?.setOnKeyListener { v, keyCode, event ->
+            if (keyCode == KeyEvent.KEYCODE_ENTER && event.action == KeyEvent.ACTION_UP && chatSearchInput?.text.toString().isNotEmpty()) {
```

Comment:
v? please make it view

Reply:
This are default variables created by the systems. So I usually prefer to keep it same as generated.